examples/cahnhilliard.py

```diff
@@ -114,7 +114,7 @@ def main(size: Length = parse('10cm'),
 
     For this reason, the `stab` enum in this script defines the stabilizing term
     `δψ`, rather than `f`, allowing Nutils to construct the residuals through
-    automatic differentiation using the `optimize` method.
+    automatic differentiation.
 
```

Comment:
Unrelated to the PR, but we're using symbolic differentiation, not automatic differentation.

Reply:
fixed